### DISCUSSION TITLE
feat(EditorForm): Add this (ReadOnlyAttribute) to mark read only

### DISCUSSION THREAD

**Discussion by wenlino** [dotnetcore/BootstrapBlazor#4764] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

EditorForm component . Add this (ReadOnlyAttribute) to mark read only

```
[ReadOnly(true)]
public string Id {get;set;}
```

### Describe the solution you'd like

EditorForm component . Add this (ReadOnlyAttribute) to mark read only

**Comment by bb-auto** [comment]
@wenlino Thank you for contacting us. We will give feedback later.
